components/transaction-modal.tsx

```diff
@@ -99,7 +101,7 @@ export default function TransactionModal({
   }
 
   function getEtherScanHref(tx: string) {
-    return `//${publicRuntimeConfig?.metaMask?.blockScanUrl}/tx/${tx}`;
+    return `//${settings?.urls?.blockScan}/tx/${tx}`;
```

Comment:
we found out that this hardcoded `tx` might screw _other_ blockscanners -- best to _not_ include that hardcoded string and just have the value on the database have it